javascript/javascript1/week2/homework4/studentManager.js

```diff
@@ -0,0 +1,43 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === '' || studentName === undefined) {
+        return `A student name is missing.`;
+      }
+    
+      if (class07Students.includes(studentName)) {
+        return `${studentName} is already in the class.`;
+      }
+    
+      if (class07Students.length >= 7 && studentName !== 'Margarethe') {
```

Comment:
Very nice and straightforward implementation! 🤩 I do think our other Queen, Mary, would be bummed out if she tried to join a full class though. 🥺😛

Reply:
I added her Highness Queen Mary to the exception rule alongside the Queen-mother Margarethe II. May the monarchs and the royalists be happy now 😄😄